cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -712,8 +616,8 @@ function LogViewerController ($scope, $window, LogViewerStore, myLogsApi, LOGVIE
     // Whenever we change the log level filter, the data needs
     // to start from scratch
     vm.data = [];
-    vm.fromOffset = -10000 + '.' + vm.startTimeMs;
-
+    // And so does the offset!!
+    vm.fromOffset = null;
```

Comment:
is this alright? I think i recall some problem before if the `fromOffset` was null, that's why we did the `-10000.<some start time>`

Reply:
@elfenheart The reason why this was changed was, when user changes the log level we used to use the start time from the last fetched log (line 168) and so the new polling mechanism doesn't retrieve any new logs. 
Talked with @poornachandra and @gsps1 regarding removing the fromoffset for the first request and the backend seems to be returning the latest 1 hour logs when no offset is mentioned. 

I can revert this back if you think this was causing some other side effect like UI getting less logs in last 1 hour and we no longer make any new request etc.,